lib/core/test/time_point_test.cpp

```diff
@@ -51,5 +50,6 @@ TEST(TimePointTypeTest, is_pod) {
   // such as sc.empty are fast and do not call constructors and initialize
   // memory --- in particular since this is also used internally for
   // initialization the output in the implementation of many operations.
-  ASSERT_TRUE(std::is_pod_v<time_point>);
```

Comment:
`is_pod` has been deprecated: https://stackoverflow.com/questions/48225673/why-is-stdis-pod-deprecated-in-c20